src/main/java/com/kainos/ea/model/JobRole.java

```diff
@@ -0,0 +1,35 @@
+package com.kainos.ea.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class JobRole {
+
+    private int id;
+    private String role_title;
+
+    @JsonCreator
+    public JobRole(
+            @JsonProperty("id") int id,
+            @JsonProperty("Kainos_Job_Title") String role_title) {
```

Comment:
Can you figure out the source of this name and fix it everywhere? It doesn't conform to naming standards

Reply:
refactored